File: python/평균.py
n = int(input())
a = list(map(int, input().split()))

# 가장 큰 점수를 구한다
m = max(a)

c = []
for i in a:
    c.append((i/m)*100)

avg = sum(c)/n

# avg = round(avg, 6)
print(avg)

File: cpp/class1/평균.cpp
#include <iostream>
using namespace std;

int n;

int main()
{
   cin >> n;
   double arr[1000] = {};
   double max, sum = 0;

   for (int i = 0; i < n; i++)
   {
      cin >> arr[i];
      if (max < arr[i])
      {
         max = arr[i];
      }
   }

   for (int i = 0; i < n; i++)
   {
      arr[i] = arr[i] / max * 100.0;
      sum += arr[i];
   }

   cout << sum / (double)n;

   return 0;
}